crates/miden-tx/src/host/account_procedures.rs

```diff
@@ -38,6 +41,21 @@ impl AccountProcedureIndexMap {
         Ok(Self(result))
     }
 
+    /// Builds an [`AccountProcedureIndexMap`] for the specified transaction inputs and arguments.
+    ///
+    /// The resulting instance will contain a map of account code into a list of procedures for the
+    /// executor account as well as any foreign accounts involved in the transaction.
+    pub fn from_transaction_params(
```

Comment:
nit: "a map of account code into a list of procedures" is not really clear. What is supposed to say that we map every code commitment to a list of procedures?

Reply:
Expanded these docs